Testing/Rails.md

```diff
@@ -0,0 +1,16 @@
+## Testing — Ruby on Rails
+
+### Different testing tools
+There are a handful of tools that you can use to test your Rails app. These are a couple that we have used at SfG:
+
+### Minitest
+This is the testing suite that comes with Rails by default. You can check out the documentation for working with Minitest and Rails [here](https://guides.rubyonrails.org/testing.html).
+
+### Basic RSpec
+This is a testing suite you can add to Rails with this [gem](https://github.com/rspec/rspec-rails#rspec-rails--).
+[Here](https://rspec.info/documentation/) are RSpec's official docs
+1. Here's a [detailed introductory tutorial](https://medium.com/craft-academy/introduction-to-ruby-and-rspec-135da4051802)
+1. [Here's an RSpec Tutorial](https://semaphoreci.com/community/tutorials/getting-started-with-rspec) (Don’t write/type anything until you reach “Setting Up Rspec”, then code along)
+
+#### Some RSpec related tooling
+1. Using Factory Bot makes testing easier. [Here](https://semaphoreci.com/community/tutorials/working-effectively-with-data-factories-using-factorygirl) and [here](https://medium.com/@prstnj/factorybot-82da45c53a1) are a couple of Factory Bot tutorials.
```

Comment:
`factories-using-factorygirl` don't love that this URL slug changed from `factorybot` back to `factorygirl` 😑 but it's broken otherwise sigh